profiler/snippets/snippets.go

```diff
@@ -28,7 +29,12 @@ func main() {
 		ServiceVersion: "1.0.0",
 		// ProjectID must be set if not running on GCP.
 		// ProjectID: "my-project",
-	}); err != nil {
+	},
+		// For OpenCensus user for Cloud Trace and Cloud Monitoring:
+		// This option prevents the Profiler agent to send Span data
+		// when the agent send profile data to the backend.
```

Comment:
A couple minor wording suggestions:
```suggestion
		// For OpenCensus users that use Cloud Trace and Cloud Monitoring:
		// cloud.google.com/go/profiler creates its own spans when making its
		// own requests to send profiling data to the server, which can be
		// noisy. This option prevents the profiler package from adding spans
		// for its own requests.
```

Reply:
Thank you for the correction! Accepted the suggestion.